requirements-dev.txt

```diff
@@ -3,3 +3,7 @@ coverage==4.5.3
 pytest-cov==2.6.1
 pytest==4.3.1
 pytest-asyncio==0.10.0
+sphinx==2.1.2
+rinohtype
```

Comment:
it might be better to set specific versions, `rinohtype` is in [beta](https://rinohtype.readthedocs.io/en/latest/) and breaking changes can occur in the future